src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -382,6 +382,51 @@ namespace JSONHelpers
         SaveFancyZonesData();
     }
 
+    bool FancyZonesData::IsAnotherWindowOfApplicationInstanceZoned(HWND window) const
```

Comment:
This works if the main window is zoned, but it won't work if the main window is not zoned, allowing to zone a child window.
We may consider merging this anyway and fix it later.

Reply:
I don't understand this comment. 
Could you clarify what it's about, maybe with test case steps that exposes this problem?

Reply:
Open Outlook, don't zone it, create a new message, zone the new message window.
The new message windows is zoned and the zone is added to the history.

Reply:
Now it's clear. Thanks.
I was aware of that when writing that code, but I couldn't figure out any reliable way to different main window and new message window. 
As new message window is created independently from main and isn't marked as child window